tests/fixtures/noqa.py

```diff
@@ -61,7 +61,7 @@ class Nested:  # noqa: Z306,Z431
 assert 2 > octal_number  # noqa: Z309
 
 
-for symbol in 'abc':  # noqa: Z224
+for symbol in 'abc':  # noqa: Z436
```

Comment:
Please, also update `tests/checkers/test_noqa.py`. There should be a new error and a count for it.